components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,212 @@
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is no user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, statusMessage] = license ? getSubscriptionLevel(license) : defaultMessage();
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card>
+                        {licenseLevel}
+                        {paid}
+                        <div className="mt-4 font-semibold text-sm">Available features:</div>
+                        <div className="flex flex-col items-start text-sm">
+                            {features &&
+                                features.map((feat: string) => (
+                                    <span className="inline-flex space-x-1">
+                                        {featureList?.includes(feat) ? (
+                                            <CheckSvg fill="currentColor" className="self-center mt-1" />
+                                        ) : (
+                                            <XSvg fill="currentColor" className="self-center h-2 mt-1" />
+                                        )}
+                                        <span>{capitalizeInitials(feat)}</span>
+                                    </span>
+                                ))}
+                        </div>
+                    </Card>
+                    <SolidCard>
+                        <div className="my-2">{statusMessage}</div>
+                        <p className="dark:text-gray-500 font-semibold">Registered Users</p>
+                        <span className="dark:text-gray-300 text-lg">{license?.userCount || 0}</span>
+                        <span className="dark:text-gray-500 text-gray-400 pt-1 text-lg"> / {userLimit} </span>
+                        <p className="dark:text-gray-500 pt-2 font-semibold">License Type</p>
+                        <h4 className="dark:text-gray-300 text-lg">{capitalizeInitials(license?.type || "")}</h4>
+                        <a
+                            className="gp-link flex flex-row mr-2 justify-end font-semibold space-x-2 mt-6"
+                            href="https://www.gitpod.io/self-hosted"
+                            target="_blank"
+                        >
+                            <span>Compare Plans</span>
```

Comment:
**suggestion**: What do you think of using text-sm here or the parent <a> element so that link text is not larger than the rest of the information on this card as also match the line height to the height of the icon used, `20px`?

```suggestion
                            <span className="text-sm">Compare Plans</span>
```

| BEFORE | AFTER |
|-|-|
| <img width="321" alt="link-before" src="https://user-images.githubusercontent.com/120486/165717850-d3ca7edd-700c-4ca9-812a-a051b831fc2e.png"> | <img width="321" alt="link-after" src="https://user-images.githubusercontent.com/120486/165717854-dd3963c9-11cf-4dec-a3f7-38cfb1f900d6.png"> |

Reply:
love it!